ServiceRequest/ServiceRequest-NonWGSTestOrderForm-HaemOncology-Example.json

```diff
@@ -0,0 +1,90 @@
+{
+  "resourceType": "ServiceRequest",
+  "id": "ServiceRequest-NonWGSTestOrderForm-HaemOncology-Example",
+  "extension": [
+    {
+      "url": "https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-Coverage",
+      "valueCodeableConcept": {
+        "coding": [
+          {
+            "system": "https://fhir.hl7.org.uk/CodeSystem/UKCore-FundingCategory",
+            "code": "nhs",
+            "display": "NHS"
+          }
+        ]
+      }
+    }
+  ],
+  "status": "active",
+  "intent": "order",
+  "category": [
+    {
+      "coding": [
+        {
+          "system": "https://fhir.hl7.org.uk/CodeSystem/UKCore-GenomeSequencingCategory",
+          "code": "cancer-non-wgs",
+          "display": "Cancer - Non-WGS"
+        }
+      ]
+    }
+  ],
+  "priority": "routine",
+  "code": {
+    "coding": [
+      {
+        "system": "https://fhir.nhs.uk/CodeSystem/England-GenomicTestDirectory",
+        "code": "M91",
+        "display": "Acute Lymphoblastic Leukaemia"
+      }
+    ]
+  },
+  "subject": {
+    "reference": "Patient/Patient-PatrickSammy-Example",
+    "identifier": {
+      "system": "https://fhir.nhs.uk/Id/nhs-number",
+      "value": "9449307873"
+    }
+  },
+  "authoredOn": "2023-11-13",
+  "requester": {
+    "reference": "PractitionerRole/PractitionerRole-HazelSmithPathology-Example"
+  },
+  "reasonCode": [
```

Comment:
The BM Aspirate should come under a sample type, not reason code. the reason code should be "New Diagnosis" (if a SNOMED CT concept can't be found this should be added as "text")